src/labels/CustomLabels.labels

```diff
@@ -1,6 +1,174 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomLabels xmlns="http://soap.sforce.com/2006/04/metadata">
     <labels>
+        <fullName>RP_ChangesGitHub</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>RP_ChangesGitHub</shortDescription>
```

Comment:
we need the shortDecription field to actually be a sentence to help our translators know where the label is used and what for.  It's unfortunate that Salesforce Setup copies <fullname> and <shortDescription> between each other in the ui.  We are guilty of doing this same thing, but now try to always put real descriptions in the field to help with translation context.